configuration/src/main/kotlin/com/malinskiy/marathon/config/serialization/ConfigurationFactory.kt

```diff
@@ -112,8 +112,8 @@ class ConfigurationFactory(
                 // Default value for analyticsTracking / bugsnagReporting in both CLI and marathon / gradle config is true.
                 // If analyticsTracking / bugsnagReporting is set to false in either CLI or marathon / gradle config, it will be disabled.
                 // Note that it's not possible to set the CLI value when running marathon via gradle.
-                analyticsTracking = !(analyticsTracking == false || configuration.analyticsTracking == false),
-                bugsnagReporting = !(bugsnagReporting == false || configuration.bugsnagReporting == false)
+                analyticsTracking = run {if (analyticsTracking == false || configuration.analyticsTracking == false) false else true},
```

Comment:
why `run` is used here? =)

Reply:
Because without it, I'm getting a compilation error: `Type mismatch: inferred type is () -> Boolean but Boolean was expected`  (I'm a completely newbie to Kotlin, so if there's a more proper way to fix the issue, feel free to suggest it!

Reply:
No worries, I've fixed it for you.